gpii/node_modules/flowManager/src/UntrustedFlowManager.js

```diff
@@ -44,16 +44,54 @@ fluid.defaults("gpii.flowManager.untrusted", {
         proximityTriggered: {
             gradeNames: "gpii.flowManager.untrusted.stateChangeHandler"
         }
+    },
+    invokers: {
+        setSettings: {
```

Comment:
We should probably partner this with a "notImplemented" invoker in FlowManager.local, just to remind ourselves to implement this at the point we ever try using a trusted FlowManager again